homeassistant/components/demo/device_tracker.py

```diff
@@ -1,7 +1,7 @@
 """Demo platform for the Device tracker component."""
 import random
 
-from homeassistant.components.device_tracker import DOMAIN
+DOMAIN = "demo"
```

Comment:
Import this

Reply:
Maybe we should call the service new? Like demo.device_tracker

Reply:
First comment is resolved. I am open to service name changes, my primary concern here was updating the domain